three.js/Makefile

```diff
@@ -17,7 +18,8 @@ minify-main: build-main
 build-lean:
 	echo > build/ar.lean.js
 	cat vendor/jsartoolkit5/build/artoolkit.min.js \
-		vendor/jsartoolkit5/js/artoolkit.api.js >> build/ar.lean.js
+		vendor/jsartoolkit5/js/artoolkit.api.js \
+		vendor/jsartoolkit5/js/artoolkit.worker.js 	>> build/ar.lean.js
```

Comment:
see this [comment](https://github.com/jeromeetienne/AR.js/pull/674#issuecomment-568770206)